public_desktop/desktop-app.js

```diff
@@ -92,6 +94,42 @@ function startDesktopApp() {
 
 	debug = gGebug( 'desktop:browser' );
 
+	try {
+		var buildEditorContextMenu = electron.remote.require( '../desktop/lib/menu/editor-context-menu' );
+		var buildGeneralContextMenu = electron.remote.require( '../desktop/lib/menu/general-context-menu' );
```

Comment:
Agree, also since it doesn't work in Windows. How do we set NODE_PATH within Electron?